README.md

```
@@ -73,6 +73,49 @@ Here is an example docker-compose.yml file:
 
 ```yaml
 services:
+<<<<<<< ours
+	n8n:
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)